scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/ModelData.scala

```diff
@@ -125,8 +128,6 @@ case class ClassLoaderModelData private (
     override val additionalConfigsFromProvider: Map[DesignerWideComponentId, ComponentAdditionalConfig],
     // This property is for easier testing when for some reason, some jars with ComponentProvider are
     // on the test classpath and CPs collide with other once with the same name.
-    // E.g. we add liteEmbeddedDeploymentManager as a designer provided dependency which also
```

Comment:
I would remove these functions now and whole comment block

Reply:
I wanted to remove it but there is still some problem with ProcessConfigCreator implementations. Nevertheless, the lite Embedded deployment manager is not the problem anymore.